ui/packages/catalog-ui-search/src/main/webapp/component/singletons/metacard-definitions.js

```diff
@@ -147,42 +189,12 @@ define([
             var definition = this.metacardTypes[id];
             return definition ? definition.alias || id : id;
         },
+        getMetacardStartingTypes: function(){
+            return metacardStartingTypes;
+        },
         metacardDefinitions: [],
         sortedMetacardTypes: [],
-        metacardTypes: {
-            anyText: {
-                id: 'anyText',
-                type: 'STRING',
-                multivalued: false
-            },
-            anyGeo: {
-                id: 'anyGeo',
-                type: 'LOCATION',
-                multivalued: false
-            },
-            'metacard-type': {
-                id: 'metacard-type',
-                type: 'STRING',
-                multivalued: false,
-                readOnly: true
-            },
-            'source-id': {
-                id: 'source-id',
-                type: 'STRING',
-                multivalued: false,
-                readOnly: true
-            },
-            cached: {
-                id: 'cached',
-                type: 'STRING',
-                multivalued: false
-            },
-            'metacard-tags': {
-                id: 'metacard-tags',
-                type: 'STRING',
-                multivalued: true
-            }
-        },
+        metacardTypes: mergeObject({}, metacardStartingTypes),
```

Comment:
✏️ I think `_.extendOwn({}, metacardStartingTypes)` would offer equivalent functionality to what you're doing here

[Underscore.extend](http://underscorejs.org/#extend)
[Underscore.extendOwn](http://underscorejs.org/#extendOwn)